generators/app/templates/src/redux/middlewares/analyticsMiddleware.js

```diff
@@ -18,6 +18,8 @@ const eventsTrackingMiddleware = getState => next => action => {
     case StackActions.RESET:
       trackNavigation(action.actions[action.index].routeName);
       break;
+    default:
+      break;
```

Comment:
What do you think if we replace this `switch` with an obj?

Reply:
@guidoprinc

Reply:
mmmm both options are great. But i must tell that if we update to react-navigation v5 this tracking will not be useful because this actions will not work anymore.

Reply:
Yeahh, i'm having some troubles with navigation v5 on facu's training. I think that we could use it but we need to sit down and discuss it because the change is really big